### DISCUSSION TITLE
ODBC error when setting up 0.7

### DISCUSSION THREAD

**Discussion by simonarbel** [microsoft/finops-toolkit#1174] [Q&A] [answered]
Deployment was successful however getting this power BI error in a customer env when trying to fetch data:

CostDetails
OLE DB or ODBC error: [Expression.Error] We cannot apply operator >= to types List and Number..
Prices
Load was cancelled by an error in loading a previous table.
Query parameters
Storage URL
Load was cancelled by an error in loading a previous table.
RangeStart
Load was cancelled by an error in loading a previous table.
RangeEnd
Load was cancelled by an error in loading a previous table.
Number of Months
Load was cancelled by an error in loading a previous table.

**Comment by lundejd** [comment]
Is your RangeStart and RangeEnd type "Date/Time" and your Number of Months type "Decimal Number" in the .pbix? 

e.g. highlight Number of Months and click Manage Parameter. Check type. 

Whenever we upgrade and I have to re-create these I forget to change from text and get a similar error.

**Reply by simonarbel** [reply]
Yes the parameters definitions are correct. Any other cause?

**Comment by MattZo1** [comment, marked as answer]
I get the same error whenever I try to use RangeEnd and RangeStart. As long as they are blank works fine.  Putting any date and time combo in results in the above error.

**Reply by flanakin** [reply]
Yes, please refrain from adding RangeStart and RangeEnd parameters if you can. I can't seem to replicate the issue due to a confusing mix of layered region formats across PBI and Windows. I haven't been able to find the issue. If someone who has PBI and M languag experience has some time, we could definitely use the help in investigating. In the meantime, avoid using RangeStart and RangeEnd. They're only provided to support incremental refresh.
